hazelcast/proxy/queue.py

```diff
@@ -31,7 +31,18 @@ class Full(Exception):
 
 
 class Queue(PartitionSpecificProxy):
+    """
+    Concurrent, blocking, distributed, observable queue. The Queue is not a partitioned data-structure. All the content
+    of the Queue is stored in a single machine (and in the backup). The Queue will not scale by adding more members
```

Comment:
observable queue. Queue is not a partitioned data structure. All of the queue content.....Queue will not....